docs/content/stable/explore/change-data-capture/debezium-connector-yugabytedb.md

```diff
@@ -35,6 +35,26 @@ Debezium supports databases with UTF-8 character encoding only. With a single-by
 
 {{< /tip >}}
 
+## Connector compatibility
+
+The connector is compatible with the following versions of YugabyteDB.
+
+| YugabyteDB | Connector |
+| :--- | :--- |
+| 2.14 (EA) | 1.9.5.y.3 |
+| 2.16 | 1.9.5.y.24 |
+| 2.18.2 | 1.9.5.y.33.2 |
+| 2.20 | 1.9.5.y.220.2 |
```

Comment:
@vaibhav-yb what is compatibility for 2024.1

Reply:
The same connector `1.9.5.y.220.2` would work for `2024.1` - but we will be releasing a new connector soon, probably on the same commit but with a newer version just to keep our version numbers inline.